addons/mail/static/src/core/web/chatter.xml

```diff
@@ -93,7 +93,12 @@
             </div>
             <t t-if="state.thread.composer.type">
                 <t t-if="state.thread.composer.type === 'message'">
-                    <div class="flex-shrink-0 px-3 pt-3 text-truncate small" style="margin-left:48px" t-out="toFollowersText"/>
+                    <div class="flex-shrink-0 px-3 pt-3 text-truncate small" style="margin-left:48px" t-if="state.thread.followers.length > 0">
```

Comment:
Nit-picking: it's more readable when `t-if` is the 1st attribute, so it's very apparent that this node presence is conditioned

Reply:
I have moved this condition to the parent `t-if` :)